source/Ship.cpp

```diff
@@ -2234,8 +2234,10 @@ void Ship::DoGeneration()
 		// 4. Shields of carried fighters
 		// 5. Transfer of excess energy and fuel to carried fighters.
 
-		const double hullAvailable = attributes.Get("hull repair rate")
-			* (1. + attributes.Get("hull repair multiplier"));
+		const double hullAvailable = (attributes.Get("hull repair rate")
+			+ (attributes.Get("hot hull repair") * Heat())
```

Comment:
I'd store heat in a variable since we're using it 4 times, counting the ones below with shields